SICAR/sicar.py

```diff
@@ -62,6 +64,38 @@ def __init__(
         self._create_session(headers=headers)
         self._initialize_cookies()
 
+    @staticmethod
+    def _parse_update_date_html(html_response) -> Dict:
+        """
+        Parse raw html getting states and updated at date.
+
+        Parameters:
+            html_response (bytes): The byte string containing html page
+
+        Returns:
+            Dict: A dict containing state sign as keys and parsed update date as value.
+
+        """
+        html_content = html_response.decode("utf-8")
+
+        soup = BeautifulSoup(html_content, "html.parser")
+
+        state_dates = {}
+
+        for state_block in soup.find_all("div", class_="listagem-estados"):
+            button_tag = state_block.find(
+                "button", class_="btn-abrir-modal-download-base-poligono"
+            )
+            state_sign = button_tag.get("data-estado") if button_tag else None
```

Comment:
What do you think if we return the objest State instead of string

```suggestion
            state_sign = State(button_tag.get("data-estado").upper()) if button_tag else None
```

P.S. The suggestion can except so more validation is required

Reply:
oh, I didnt see this one, It would make sense for sure to keep consistency.
I will provide for tomorrow so wait a sec for your next review

Reply:
I made this change and feels like the dict keys are a little scary at first glance but it is easily interchangeable to the string of the state if one would like.
Overall I think it is a good choice because by enforcing the use of State(state_sign) we would cross check that the state exists for sure, so if any misspelling occurs it would be noticeable.